INSTALL

```diff
@@ -540,8 +540,9 @@ The specific libraries and their options are:
   See also the "Specific options for hypre" section at the end of this file.
   URL: https://github.com/hypre-space/hypre and https://www.llnl.gov/casc/hypre
   Options: HYPRE_OPT, HYPRE_LIB.
-  Versions: HYPRE >= 2.10.0b,
-            HYPRE >= 2.20.0 for '--enable-mixedint' support.
+  Versions: HYPRE >= 2.10.0b  (HYPRE built without CUDA)
+            HYPRE >= 2.20.0   (for '--enable-mixedint' support.)
+            HYPRE >= 2.22.0   (HYPRE built with CUDA)
```

Comment:
Are all of these versions still correct?

Reply:
Yes, I think so. The only caveat is that ADS+CUDA needs HYPRE > 2.22.0.

Reply:
> The only caveat is that ADS+CUDA needs HYPRE > 2.22.0.

I think that's ok